src/sections/safe.tsx

```diff
@@ -8,21 +7,117 @@ import { useAppDispatch, useAppSelector } from '../store';
 import Section from '../future-hopr-lib-components/Section';
 import { useSigner } from '../hooks';
 import { actionsAsync } from '../store/slices/safe/actionsAsync';
-
+import { utils } from 'ethers';
 function SafeSection() {
   const dispatch = useAppDispatch();
-  const safe = useAppSelector((store: Store) => store.safe);
+  const safe = useAppSelector((store) => store.safe);
   const { signer } = useSigner();
 
   useEffect(() => {
     if (signer) {
       dispatch(actionsAsync.getSafesByOwnerThunk({ signer }));
     }
   }, [signer]);
+
   return (
     <Section className="Section--safe" id="Section--safe" yellow>
-      SAFE REDUX STORE
+      <h1>SAFE REDUX STORE</h1>
```

Comment:
Just a comment, when we work on the UI we might want to handle uppercase text with styles `text-transform: uppercase;`  :)

Reply:
yeah the uppercase was also not part of this pr